CHANGELOG.md

```diff
@@ -4,7 +4,27 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v6.4.0](https://github.com/voxpupuli/puppet-systemd/tree/v6.4.0) (2024-02-24)
+## [v6.5.0](https://github.com/voxpupuli/puppet-systemd/tree/v6.5.0) (2024-03-05)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-systemd/compare/v6.4.0...v6.5.0)
+
+**Implemented enhancements:**
+
+- Add possibiliry to setup limits for user sessions [\#417](https://github.com/voxpupuli/puppet-systemd/issues/417)
+- add a cron-like systemd::timer interface [\#374](https://github.com/voxpupuli/puppet-systemd/issues/374)
+- Support `[Slice]` in manage\_unit and manage\_dropin [\#420](https://github.com/voxpupuli/puppet-systemd/pull/420) ([traylenator](https://github.com/traylenator))
+- feat: add a cron like wrapper for timers [\#419](https://github.com/voxpupuli/puppet-systemd/pull/419) ([TheMeier](https://github.com/TheMeier))
+
+**Closed issues:**
+
+- Running the example code for `systemd::manage_unit` from REFERENCE.md causes an error:  [\#414](https://github.com/voxpupuli/puppet-systemd/issues/414)
```

Comment:
I addes the `skip-changelog` label here because we already have the related PR in the changelog.